### ISSUE TITLE
5 articles

### ISSUE THREAD

**Issue by parsaanooshfar-lab** [parsaanooshfar-lab/homework-machine-learning#8]
1)https://link.springer.com/article/10.1007/s10916-024-02089-5?utm_source=chatgpt.com
چکیده: این مرور به نقش فناوری‌های هوش مصنوعی و یادگیری ماشینی در مقابله با مقاومت میکروبی دارویی
می‌پردازد
2)https://www.sciencedirect.com/science/article/abs/pii/S2666991924000198?utm_source=chatgpt.com
چکیده: مرور سیستماتیک کاربردهای یادگیری ماشینی در پیش‌بینی مقاومت میکروبی دارویی
3)https://doi.org/10.3390/antibiotics14020134
چکیده: مرور جامع دربارهٔ مدل‌های هوش مصنوعی و یادگیری ماشینی در برنامه‌های مدیریت مصرف آنتی‌بیوتیک  و مقاومت میکروبی
4)https://www.frontiersin.org/journals/artificial-intelligence/articles/10.3389/frai.2024.1479855/full?utm_source=chatgpt.com
چکیده: مرور شرایط، روندها، چالش‌ها و فرصت‌های به‌کارگیری 
ML و DL در مکزیک
5)https://doi.org/10.1371/journal.pone.0319460
چکیده: مرور سیستماتیک کاربردهای 
ML در پیش‌بینی مقاومت میکروبی برای پاتوژن‌های بحرانی در محیط‌های واقعی درمانی

1. From Data to Decisions: Leveraging AI and ML in Combating Antimicrobial Resistance

روش‌های ML استفاده شده:

Random Forest (RF): برای پیش‌بینی مقاومت آنتی‌بیوتیکی

Support Vector Machines (SVM): طبقه‌بندی مقاومت/حساسیت

Gradient Boosting / XGBoost: برای افزایش دقت پیش‌بینی

Neural Networks (NN): در برخی مدل‌های پیچیده و چند لایه

2. Machine Learning to Predict Antimicrobial Resistance (Kherabi et al.)

روش‌های ML استفاده شده:

Decision Tree: برای تشخیص الگوریتمی مقاومت دارویی

Random Forest: برای پیش‌بینی دقیق‌تر

Logistic Regression: برای پیش‌بینی ریسک مقاومت

SVM و k-Nearest Neighbors (k-NN): برای ارزیابی کارایی مدل‌ها

3. The Role of AI and ML Models in Antimicrobial Stewardship in Public Health (Pennisi et al.)

روش‌های ML استفاده شده:

Random Forest و Decision Trees: برای پیش‌بینی مقاومت

Support Vector Machines: برای طبقه‌بندی داده‌ها

Neural Networks (Deep Learning): برای الگوهای پیچیده در داده‌های بالینی

Ensemble Methods (Boosting, Bagging): برای افزایش دقت پیش‌بینی

4. A Systematic Review of ML and DL Approaches in Mexico (Uc Castillo et al.)

روش‌های ML/DL استفاده شده:

Random Forest و Gradient Boosting: برای پروژه‌های بزرگ داده

Deep Learning / Convolutional Neural Networks (CNN): در تحلیل داده‌های پیچیده

k-NN و SVM: برای مسائل طبقه‌بندی سنتی

Federated Learning: برای پردازش داده‌های توزیع‌شده و حفظ حریم خصوصی

5. ML for Predicting Antimicrobial Resistance in Critical Pathogens (PLOS ONE)

روش‌های ML استفاده شده:

Random Forest و Decision Tree: پیش‌بینی مقاومت پاتوژن‌ها

Gradient Boosting / XGBoost: دقت بالا در پیش‌بینی

Neural Networks / Deep Learning: تحلیل داده‌های پیچیده بالینی

Logistic Regression و Ensemble Methods: تحلیل‌های مقایسه‌ای و بهبود عملکرد مدل
